components/Comments/CommentForm.vue

```diff
@@ -76,22 +87,38 @@
       }
     },
     methods: {
+      cancel (form) {
+        if (this.isCommentFormOfContribution) {
+          form.content = ''
+        } else {
```

Comment:
Please use if else and try to avoid unneeded multi level nesting and use curly brackets (see js standard styling)